### ISSUE TITLE
Function to classify malignant/non-malignant cells 

### ISSUE THREAD

**Issue by rgranit** [icbi-lab/infercnvpy#11]
Have you considered adding a function that would classify cells into malignant/non-malignant similar to [inferCNA 'findMalignant' function](https://github.com/jlaffy/infercna/blob/master/R/findMalignant.R)?

**Comment by grst** [comment]
This looks interesting, thanks for the link! 

As far as I understood they essentially compute the correlation between CNV profiles for each cell and try to fit a bimodal distribution (where one mode represents malignant cells and the other mode normal cells)? 

I'm wondering how well that works... I have seen quite strong cell-type biases as well in the datasets I ran infercnvpy on.

**Comment by rgranit** [comment]
Yes, I believe your description of the function is correct. I've attempted to reproduce the plots that they generate using infercnvpy:

```
adata.obs["cnv_corr"] = 0
df=pd.DataFrame.sparse.from_spmatrix(adata.obsm['X_cnv'])
adata.obs["cnv_corr"] = df.corrwith(df.mean(),axis=1).values
```

This is how the plot looks like: 

![image](https://user-images.githubusercontent.com/9028967/116998657-3ade4480-ace7-11eb-8568-4a73260d1e9e.png)

In red I've colored cells which I believe to be tumor. Next I've used GaussianMixture to try and find bimodal clusters and worked reasonably well for some datasets, but not all and in assumes n=2 which is not always the case.

Happy for any thoughts or inputs
